integration_tests/src/main/python/join_test.py

```diff
@@ -841,3 +841,28 @@ def do_join(spark):
             "spark.sql.adaptive.enabled": aqeEnabled,
             "spark.sql.autoBroadcastJoinThreshold": bhjThreshold
         })
+
+
+@ignore_order
+@pytest.mark.parametrize('aqeEnabled', [True, False], ids=['AQE_ON', 'AQE_OFF'])
+def test_existence_join_in_broadcast_nested_loop_join(spark_tmp_table_factory, aqeEnabled):
```

Comment:
This is unlikely to test the handling of splitting the join output due to batch size limits.  There should be a test, either as another dimension of this test or a separate test, that will cause output batches to be split.

Reply:
Emit the batch directly as the `exists` boolean column is small.
No need to test the splitting.